cloudbot/util/web.py

```diff
@@ -59,7 +59,11 @@ def expand(url, service=None):
     return impl.expand(url)
 
 
-def paste(data, ext='txt', service=DEFAULT_PASTEBIN):
+class NoPasteException(Exception):
```

Comment:
Why is this in this PR?

Reply:
Because I added it specifically for the use of this plugin (for now)